### PR TITLE
Implement continue pooling

### PR DESCRIPTION
**Pull request by elopezanaya**
# PR Details

### Description
_Include a description of the problem to be solved_
In some scenarios, customer states first message from copilot is "lost", it seems this is due to an off-sync when websocket is initializing and pooling stops.

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

- Enable Pooling to continue for 10 secs after websocket has initialized, as backup , to prevent messages in transit being lost
- When endchat is called, ensure pooling is stop as well

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Endchat in the middle of the conversation and long pool must stop
image

### Normal flow
image
#### no more pooling after session close
image

### Slow 4G
image
#### no more pooling after session close
image

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Enabling LongPooling to continue 10 secs after websocket is set, to prevent messages lost.
```
Comment (dleblondMSFT) [inline CHANGELOG.md:9]: Polling*, will need to correct this typo in a few places.

**Review by dleblondMSFT — APPROVED** [review]

**Review by mithunjohny — APPROVED** [review]
